tests/data/test_history.py

```diff
@@ -133,8 +133,8 @@ def test_load_data_with_new_pair_1min(ohlcv_history_list, mocker, caplog,
     load_pair_history(datadir=tmpdir1, timeframe='1m', pair='MEME/BTC')
     assert file.is_file()
     assert log_has_re(
-        'Download history data for pair: "MEME/BTC", timeframe: 1m '
-        'and store in .*', caplog
```

Comment:
if you remove the `.*` from the regex, the method assumes the given string is the full string - which is not the case (there's some random stuff at the end).

Reply:
yes it happend cuz of bunch of edits I done and I forgoted this star... :+1:

Reply:
@xmatthias  Not Solved :))))))))))))))